src/client/services/socketIo.service.js

```diff
@@ -0,0 +1,18 @@
+import io from 'socket.io-client';
+import { store } from 'resources/store';
+import { UPDATE_USER } from 'resources/user/user.actions';
+
+const socket = io(window.config.webSocketUrl, { transports: ['websocket'], query: { token: window.token } });
+
+socket.on('connect', () => {
+  console.log('WS connected'); //eslint-disable-line
+  socket.emit('subscribe', window.user._id);
```

Comment:
Subscribe to `user-${user._id}` room from client side instead than just `user._id`

Check corresponding comment from API starter PR:
paralect/koa-api-starter#33